hack/install-monitoring-tools.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+set -euf -o pipefail
+
+GRAFANA_PORT=3000
```

Comment:
```suggestion
GRAFANA_PORT=${GRAFANA_PORT:-"3000"}
```

Reply:
Omer had suggested a similar change. I've updated the script to allow passing the port as a variable to the make target